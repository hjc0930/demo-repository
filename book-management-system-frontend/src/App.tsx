// import Routers from "./routers";

function App() {
  return <>App</>;
  // return <Routers />;
}

export default App;

const enum WeekEnum {
  Sunday = "Sunday",
  Monday = "Monday",
  Tuesday = "Tuesday",
  Wednesday = "Wednesday",
  Thursday = "Thursday",
  Friday = "Friday",
  Saturday = "Saturday",
}
