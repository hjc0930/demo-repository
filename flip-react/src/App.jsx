import Flip2 from "./components/Flip2"

function App() {
  return <>
  {/* <Flip1 /> */}
  <Flip2 /></>
}

export default App
