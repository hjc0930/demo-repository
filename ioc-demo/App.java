import java.lang.reflect.InvocationTargetException;

public class App {
    public static void main(String[] args) throws ClassNotFoundException, NoSuchMethodException, InvocationTargetException, InstantiationException, IllegalAccessException {
        ClassLoader systemClassLoader = ClassLoader.getSystemClassLoader();

        Class<?> container = systemClassLoader.loadClass("Container");

        Object o = container.getDeclaredConstructor().newInstance();

        System.out.println(o);
    }
}
