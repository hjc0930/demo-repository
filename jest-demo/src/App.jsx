import { Button, Dropdown } from "antd";

const App = () => {
  return (
    <>
      <Dropdown menu={[{ key: "1", label: "123123" }]}>
        <Button>Click me</Button>
      </Dropdown>
      <div>
        <h1>Hello world!</h1>
      </div>
    </>
  );
};

export default App;
