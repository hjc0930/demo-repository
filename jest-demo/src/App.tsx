import { Button, Dropdown } from "antd";
import { cloneDeep } from "lodash-es";

const App = () => {
  return (
    <div>
      <div>{Object.keys(cloneDeep({ a: 1, b: 2 })).toString()}</div>
      <Dropdown menu={[{ key: "1", label: "123123" }]}>
        <Button>Click me</Button>
      </Dropdown>
      <div>
        <h1>Hello world!</h1>
      </div>
    </div>
  );
};

export default App;
