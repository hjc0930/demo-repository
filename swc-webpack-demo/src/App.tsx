import { useState } from "react";
import { add } from "./utils/add";

const App = () => {
  const [value, setValue] = useState("");
  return (
    <>
      <h1>React App</h1>
      <input
        type="text"
        value={value}
        onChange={(e) => setValue(e.target.value)}
      />
      <div>{add(1, 2)}</div>
    </>
  );
};

export default App;
