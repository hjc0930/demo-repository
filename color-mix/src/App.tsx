import "./App.css";

const getTintVariantColor = (color: string) => {
  return {
    50: `color-mix(in srgb, ${color} 5%, white)`,
    100: `color-mix(in srgb, ${color} 10%, white)`,
    200: `color-mix(in srgb, ${color} 30%, white)`,
    300: `color-mix(in srgb, ${color} 50%, white)`,
    400: `color-mix(in srgb, ${color} 70%, white)`,
    500: `${color}`,
    600: `color-mix(in srgb, ${color}, black 10%)`,
    700: `color-mix(in srgb, ${color}, black 30%)`,
    800: `color-mix(in srgb, ${color}, black 50%)`,
    900: `color-mix(in srgb, ${color}, black 70%)`,
  };
};

function App() {
  const result = getTintVariantColor("#96454c");

  return (
    <ul>
      {Object.entries(result).map(([key, value], index) => (
        <li
          style={{
            backgroundColor: value,
            color: index < 5 ? "#000000" : "#ffffff",
          }}
        >
          {key}
        </li>
      ))}
    </ul>
  );
}

export default App;
