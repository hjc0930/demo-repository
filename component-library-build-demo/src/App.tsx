import logo from "./logo.svg";
import { Button } from "@hjc0930/component-demo";

import "./App.css";
import "@hjc0930/component-demo/umd/style.min.css";

const request = require.context("./components", true, /\.css$/);

const obj = {};

request.keys().forEach((fileName: any) => {
  console.log(fileName);

  obj[fileName] = request(fileName);
});

console.log(obj);

function App() {
  return (
    <div className="App">
      <Button>Ccc</Button>
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.tsx</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
    </div>
  );
}

export default App;
