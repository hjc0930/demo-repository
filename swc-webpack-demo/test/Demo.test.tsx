import { render } from "@testing-library/react";
import App from "@/App";
import { getEnvironment, getEnvironmentVariable } from "./a";

process.env.PREFIX = "/aaa";

describe("Demo", () => {
  test("App", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });

  test("getEnvironment", async () => {
    const result = getEnvironment();
    expect(result).toBe("/aaa");
  });

  test("getEnvironmentVariable", () => {
    const resultVariable = getEnvironmentVariable();
    expect(resultVariable).toBe("/aaa");
  });
});
