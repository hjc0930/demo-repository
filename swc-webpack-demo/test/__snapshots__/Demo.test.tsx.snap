// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Demo App 1`] = `
<div>
  <h1>
    React App
  </h1>
  <input
    type="text"
    value=""
  />
  <div>
    3
  </div>
</div>
`;
